### ISSUE TITLE
Feature Request: Generate different Daily Notes for different days of the week

### ISSUE THREAD

**Issue by evantkchong** [liamcain/obsidian-calendar-plugin#106]
Hi there! First off I'd like to thank you for this really great plugin! I've been using it for a while now and it's definitely made using obsidian easier.

I would like to suggest adding an option to use a different Daily Note template for different days of the week.
It would help folks like me who would like to plan their weekends separately from their weekdays.

**Comment by sroebuck** [comment]
Yes, another thank you and request for this enhancement.  As well as having very different weekends from weekdays, I also have a pattern to my weeks activities that, combined with Day Planner, means that it would be really handy to be able to create different templates for different days of the week.

**Comment by megclaypool** [comment]
This would be a neat feature! +1 😀

**Comment by konhi** [comment]
It is possible to write [Templater](https://github.com/SilentVoid13/Templater) script that changes content based on current day.
